src/features/campaign-sheet/CampaignSheetPage.tsx

```diff
@@ -79,9 +88,74 @@ export function CampaignSheetPage() {
       });
   };
 
+  const handleRemoveCampaign = async () => {
+    try {
+      await confirm({
```

Comment:
Glad you added the confirm here! I really like the ease of use of that dependency you added.